charts/airflow/files/pod_template.kubernetes-helm-yaml

```diff
@@ -46,6 +46,9 @@ spec:
     {{- if .Values.dags.gitSync.enabled }}
     {{- include "airflow.container.git_sync" (dict "Release" .Release "Values" .Values "sync_one_time" "true") | indent 4 }}
     {{- end }}
+    {{- if .Values.kubernetesPodTemplate.extraInitContainers }}
+    {{- toYaml .Values.kubernetesPodTemplate.extraInitContainers | nindent 4 }}
+    {{- end }}
```

Comment:
You also need to update line 41, so that it will enable the `initContainers` section if `.Values.kubernetesPodTemplate.extraInitContainers` is defined. (Otherwise this wont work unless `$extraPipPackages` or `.Values.dags.gitSync.enabled` are enabled)

For example:
```yaml
  {{- if or ($extraPipPackages) (.Values.dags.gitSync.enabled) (.Values.kubernetesPodTemplate.extraInitContainers) }}
  initContainers:
```

Reply:
Ah gotcha, this is resolved.